start.py

```diff
@@ -9,17 +12,19 @@
     logger.setLevel(logging.DEBUG)
     handler = logging.StreamHandler()
     formatter = logging.Formatter(
-                "[%(asctime)s] > "
-                "[%(filename)s:%(lineno)d] %(message)s"
-            )
+        "[%(asctime)s] > "
+        "[%(filename)s:%(lineno)d] %(message)s"
+    )
     handler.setFormatter(formatter)
     logger.addHandler(handler)
 
     QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling)
     app = QApplication([])
     window = QWidget()
+    window.setWindowTitle("termqt on Windows")
```

Comment:
The title should stay platform-invariant?

Reply:
Will fix it. Forgot about that.